intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -1,10 +1,29 @@
-# -*- coding: utf-8 -*-
+"""
+XMPP Collector Bot
+Connects to a XMPP Server and a Room and reads data from the room.
+If no room is provided it collects events which were sent to the xmpp user
```

Comment:
"no room" is equivalent to an empty string? I'm missing this information in the docs.